ci-operator/config/openshift/loki/openshift-loki-upstream-v3.1.0.yaml

```
@@ -0,0 +1,58 @@
+base_images:
+  base:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-openshift-enterprise-base-4.16.multi
+  ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+build_root:
+  image_stream_tag:
+    name: builder
+    namespace: ocp
+    tag: rhel-9-golang-1.21-builder-multi-openshift-4.16
+images:
+- dockerfile_path: Dockerfile.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: loki
+- dockerfile_path: Dockerfile.promtail.ocp
+  from: base
+  inputs:
+    ocp_builder_rhel-9-golang-1.21-builder-multi-openshift-4.16:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.21-openshift-4.16
+  to: promtail
+promotion:
+  to:
+  - name: v3.1.0
+    namespace: logging
+releases:
+  latest:
+    release:
+      channel: stable
+      version: "4.15"
```

shouldn't it be 4.16?
```suggestion
      version: "4.16"
```

---

@JoaoBraveCoding 4.16 stable is not out yet.

---

Afaik 4.16.0 was release June 27th? [stable](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/) already has `4.16.0`.